protocols/v2/noise-sv2/doc/encryption_guide.md

```diff
@@ -0,0 +1,154 @@
+# Noise encryption guide
```

Comment:
Do you want to merge in the nits I made on [this PR](https://github.com/jakubtrnka/stratum/pull/1)?

Reply:
Sry, I'm bit lost. I'm not used to working with github and this all forking and merging.
I don't understand what this PR is. There are 14 commits that are being requested to the main branch. Some of them are commits from my devel branch.

Reply:
If you have some suggestions for my noise_issues branch, please tell me what to do. What I should merge and where.

I'd suggest that you rebase the noise-issues-nits and open PR directly onto my noise_issues branch. If you have different workflow, you need to explain it to me :)